app/api/helpers/scheduled_jobs.py

```diff
@@ -249,22 +251,27 @@ def send_monthly_event_invoice():
             user = event.owner
             admin_info = get_settings()
             currency = event.payment_currency
-            ticket_fee_object = (
-                db.session.query(TicketFees).filter_by(currency=currency).one()
-            )
+            try:
+                ticket_fee_object = (
+                    db.session.query(TicketFees).filter_by(currency=currency).one()
+                )
+            except NoResultFound:
+                raise NotFoundError(
+                    {'source': ''}, 'Ticket Fee not set for {}'.format(currency)
+                ).respond()
             ticket_fee_percentage = ticket_fee_object.service_fee
             ticket_fee_maximum = ticket_fee_object.maximum_fee
             orders = Order.query.filter_by(event=event).all()
             gross_revenue = event.calc_monthly_revenue()
-            ticket_fees = event.tickets_sold * (ticket_fee_percentage / 100)
-            if ticket_fees > ticket_fee_maximum:
-                ticket_fees = ticket_fee_maximum
-            net_revenue = gross_revenue - ticket_fees
+            invoice_amount = gross_revenue * (ticket_fee_percentage / 100)
```

Comment:
Not sure about this

Reply:
https://www.eventbrite.com/organizer/pricing/

Reply:
Considering the Absorb Fees option as of now. Pass fees onto attendees can be implemented as well as a part of the upcoming features.

Reply:
@diru1100 Hi, would be great if you could review the logic for this. Thanks!

Reply:
Hey, Eventbrite seems to calculate it in the same way. The only difference is that they add a fixed rate to the calculated amount too. 2% + 0.79$

Reply:
@iamareebjamal I think this should be good to go.